src/Tests/Tests/QueryDsl/Specialized/DistanceFeature/DistanceFeatureQueryUsageTests.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Elastic.Xunit.XunitPlumbing;
+using Nest;
+using Tests.Core.ManagedElasticsearch.Clusters;
+using Tests.Domain;
+using Tests.Framework.EndpointTests.TestState;
+
+namespace Tests.QueryDsl.Specialized.DistanceFeature
+{
+	/**
+	* Boosts the relevance score of documents closer to a provided origin date or point. For example, you can use this query to give
+	* more weight to documents closer to a certain date or location.
+	* You can use the distance_feature query to find the nearest neighbors to a location. You can also use the query in a bool
+	* search’s should filter to add boosted relevance scores to the bool query’s scores.
+	*/
+	[SkipVersion("<7.2.0", "Implemented in version 7.2.0")]
+	public class DistanceFeatureQueryUsageTests : QueryDslUsageTestsBase
```

Comment:
Make some assertions on the response?

Reply:
Not possible, no method to override on `ApiTestBase`